CMSgov/mint-app
# [NOREF] GitHub Actions copied from EASi

# No Ticket

## Changes and Description

Copied GitHub Actions from EASi.

Jobs that cannot run due to lacking infrastructure (mainly deploy jobs, and jobs that pull/push images from ECR) have been disabled with an `if: ${{ false }}` block on the job.



## How to test this change

Not sure the best way to test this, but I'll be enabling actions on the repo starting with this PR (guessing we'll have some secrets to fill out, and will see some failing builds in the meantime)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
